pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1764,6 +1764,10 @@ func configToStatus(ctx *domainContext, config types.DomainConfig,
 			}
 			ds.Devtype = ""
 			need9P = true
+		} else if dc.Format == zconfig.Format_ISO {
+			// set required devtype and adjust format to raw as required by hypervisor
+			ds.Format = zconfig.Format_RAW
+			ds.Devtype = "cdrom"
```

Comment:
Does this also need to take types.LEGACY virtualization mode into account?

Reply:
Why not, modified